### ISSUE TITLE
Failed to in the ruby-2.3.0

### ISSUE THREAD

**Issue by arafiu-rpx** [wordtreefoundation/melisa#4]
Here is my full error trace

Building native extensions.  This could take a while...
ERROR:  Error installing melisa:
    ERROR: Failed to build gem native extension.

```
current directory: /Users/abdulr/.rvm/gems/ruby-2.3.0/gems/melisa-0.2.5/ext
```

/Users/abdulr/.rvm/rubies/ruby-2.3.0/bin/ruby -r ./siteconf20160202-13963-131f4ds.rb extconf.rb
  -- ./configure --enable-sse3 --prefix='/Users/abdulr/.rvm/gems/ruby-2.3.0/gems/melisa-0.2.5/ext/pkg'
  -- make install
**\* extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
    --with-opt-dir
    --with-opt-include
    --without-opt-include=${opt-dir}/include
    --with-opt-lib
    --without-opt-lib=${opt-dir}/lib
    --with-make-prog
    --without-make-prog
    --srcdir=.
    --curdir
    --ruby=/Users/abdulr/.rvm/rubies/ruby-2.3.0/bin/$(RUBY_BASE_NAME)
extconf.rb:7:in `sys': make install failed, please report issue on https://github.com/wordtreefoundation/melisa (RuntimeError)
    from extconf.rb:28:in`block in <main>'
    from /Users/abdulr/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/fileutils.rb:128:in `chdir'
    from /Users/abdulr/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/fileutils.rb:128:in`cd'
    from extconf.rb:26:in `<main>'

extconf failed, exit code 1

Gem files will remain installed in /Users/abdulr/.rvm/gems/ruby-2.3.0/gems/melisa-0.2.5 for inspection.
Results logged to /Users/abdulr/.rvm/gems/ruby-2.3.0/extensions/x86_64-darwin-15/2.3.0/melisa-0.2.5/gem_make.out

**Comment by arafiu-rpx** [comment]
This is the error I am getting while trying to install it manually -

make install
Making install in lib
test -z "/Users/abdulr/.rvm/gems/ruby-2.3.0/gems/melisa-0.2.5/ext/pkg/lib" || .././install-sh -c -d "/Users/abdulr/.rvm/gems/ruby-2.3.0/gems/melisa-0.2.5/ext/pkg/lib"
 /bin/sh ../libtool   --mode=install /usr/bin/install -c   libmarisa.la '/Users/abdulr/.rvm/gems/ruby-2.3.0/gems/melisa-0.2.5/ext/pkg/lib'
libtool: install: /usr/bin/install -c .libs/libmarisa.lai /Users/abdulr/.rvm/gems/ruby-2.3.0/gems/melisa-0.2.5/ext/pkg/lib/libmarisa.la
install: .libs/libmarisa.lai: No such file or directory
make[2]: **\* [install-libLTLIBRARIES] Error 71
make[1]: **\* [install-am] Error 2
make: **\* [install-recursive] Error 1

**Comment by andrassy** [comment]
I'm getting the same with Ruby 2.5.3. Did you ever fix this?

**Comment by bendangelo** [comment]
I fixed this: https://github.com/wordtreefoundation/melisa/pull/6

**Comment by pboling** [comment]
Fixed by #6
